### PR TITLE
ResultView: Guard against undefined mimeType

### LINKED ISSUE #734: Uncaught Invariant Violation: Minified React error #130; visit http://facebook.github.io/react/do...
Hi Lukas,

Thanks for the great help. This is not so crucial but technically with irkernel one is able to run ggplot2 but in hydrogen i got this error.

```
Uncaught Invariant Violation: Minified React error #130; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
events.js:160
Hide Stack Trace
Invariant Violation: Minified React error #130; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at reactProdInvariant (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/reactProdInvariant.js:31:15)
    at ReactCompositeComponentWrapper.instantiateReactComponent [as _instantiateReactComponent] (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/instantiateReactComponent.js:74:250)
    at ReactCompositeComponentWrapper.performInitialMount (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactCompositeComponent.js:367:22)
    at ReactCompositeComponentWrapper.mountComponent (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactCompositeComponent.js:258:21)
    at Object.mountComponent (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactReconciler.js:46:35)
    at mountComponentIntoNode (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactMount.js:104:32)
    at ReactReconcileTransaction.perform (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/Transaction.js:140:20)
    at batchedMountComponentIntoNode (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactMount.js:126:15)
    at ReactDefaultBatchingStrategyTransaction.perform (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/Transaction.js:140:20)
    at Object.batchedUpdates (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactDefaultBatchingStrategy.js:62:26)
    at Object.batchedUpdates (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactUpdates.js:97:27)
    at Object._renderNewRootComponent (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactMount.js:320:18)
    at Object._renderSubtreeIntoContainer (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactMount.js:401:32)
    at Object.render (/Users/mea/.atom/packages/Hydrogen/node_modules/react-dom/lib/ReactMount.js:422:23)
    at ResultView.addResult (/Users/mea/.atom/packages/Hydrogen/lib/result-view.js:193:14)
    at /Users/mea/.atom/packages/Hydrogen/lib/main.js:273:12
    at ZMQKernel.onIOMessage (/Users/mea/.atom/packages/Hydrogen/lib/zmq-kernel.js:355:7)
    at Socket.<anonymous> (/Users/mea/.atom/packages/Hydrogen/node_modules/jmp/index.js:347:17)
    at emitMany (events.js:127:13)
    at Socket.emit (events.js:201:7)
    at Socket._emitMessage (/Users/mea/.atom/packages/Hydrogen/node_modules/zeromq/lib/index.js:631:15)
    at Socket._flushRead (/Users/mea/.atom/packages/Hydrogen/node_modules/zeromq/lib/index.js:642:10)
    at Socket._flushReads (/Users/mea/.atom/packages/Hydrogen/node_modules/zeromq/lib/index.js:678:15)
    at _zmq.onReadReady (/Users/mea/.atom/packages/Hydro
```

### PR DESCRIPTION
**Pull request by n-riesco**
@rgbkrk Does this PR fix #734?

### CODE REVIEW

**Comment by n-riesco** [comment]
Travis also fails in master (see #747).

File: lib/result-view.js [hunk 1]
```diff
@@ -190,7 +190,9 @@ export default class ResultView {
     //   });
     // });
     const div = document.createElement("div");
-    ReactDOM.render(<Transform data={bundle.get(mimeType)} />, div);
+    if (mimeType) {
```
Comment (lgeiger) [inline lib/result-view.js]: I think we should move the check and  above [this line](https://github.com/nteract/hydrogen/pull/746/files#diff-375c469d3548707f41002f462a83207fR178) and completely return from the function if there's no supported mime type. What do you think?

Comment (n-riesco) [inline lib/result-view.js]: I wasn't sure if it was OK to return at that point.
I'll update the PR now.

**Comment by lgeiger** [comment]
I think we should release a new version :shipit:

**Comment by rgbkrk** [comment]
Makes me wonder if our interface could be simplified by having a null transform. It would accept props but return a null react element. 🤔

**Comment by ahsen1402** [comment]
lukas thanks for the help and your time, looking forward for the update,

**Pull request merged**